.github/workflows/binaries-nightly-release.yml

```diff
@@ -12,22 +12,21 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Setup Miniconda
-        uses: conda-incubator/setup-miniconda@v2
+      - name: Setup Python
+        uses: actions/setup-python@v4
         with:
-          miniconda-version: "latest"
           python-version: "3.10"
-
+      - name: Install Astral UV
```

Comment:
Use this GHA instead of manual installation: https://docs.astral.sh/uv/guides/integration/github/